alfalfa_web/server/advancer.js

```diff
@@ -32,20 +35,20 @@ class Advancer {
   }
 
   advance(siteRefs) {
-    return new Promise((resolve, reject) => {
+    return new Promise((resolve) => {
```

Comment:
Possibly out of scope for this PR, but it would be nice to move most of the guts of this to an API function which takes `message_name`, and `params` and provides callbacks for when a response happens (error or otherwise). The message handling on the worker side is very standard so I don't see any reason to not have message send/receive not neatly packaged.

Reply:
I considered moving this to the API class, but since it already existed and was standalone I just leveraged it as-is